<!DOCTYPE html>
<html lang="en">

<head>
  <meta http-equiv="content-type" content="text/html; charset=utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="description" content="">
  <meta name="generator" content="Hugo 0.54.0" />

  <title>Input on the Draft NIH Strategic Plan for Data Science &middot; Tristan Berchoux</title>

  
  
  <link rel="stylesheet" href="https://cdn.bootcss.com/pure/0.6.2/pure-min.css">

  <!--[if lte IE 8]>
  <link rel="stylesheet" href="https://cdn.bootcss.com/pure/0.6.2/grids-responsive-old-ie-min.css">
  <![endif]-->
  <!--[if gt IE 8]><!-->
  <link rel="stylesheet" href="https://cdn.bootcss.com/pure/0.6.2/grids-responsive-min.css">
  <!--<![endif]-->

  <!--[if lte IE 8]>
  <link rel="stylesheet" href="/css/side-menu-old-ie.css">
  <![endif]-->
  <!--[if gt IE 8]><!-->
  <link rel="stylesheet" href="/css/side-menu.css">
  <!--<![endif]-->

  <link rel="stylesheet" href="/css/blackburn.css">

  <link rel="stylesheet" href="/css/github-gist.css" rel="stylesheet" id="theme-stylesheet">
  <script src="/js/highlight.pack.js"></script>
  <script>hljs.initHighlightingOnLoad();</script>

  
  <script async src="https://use.fontawesome.com/32c3d13def.js"></script>

  
  

  
  <link rel="stylesheet" href="//cdn.bootcss.com/highlight.js/9.11.0/styles/github.min.css">
  <script src="//cdn.bootcss.com/highlight.js/9.11.0/highlight.min.js"></script>
  <script src="//cdn.bootcss.com/highlight.js/9.11.0/languages/r.min.js"></script>
  <script>hljs.configure({languages: []}); hljs.initHighlightingOnLoad();</script>
  

  <link rel="shortcut icon" href="/img/favicon.ico" type="image/x-icon" />

  
  

</head>


<body>
<div id="layout">

  
<a href="#menu" id="menuLink" class="menu-link">
  
  <span></span>
</a>
<div id="menu">

  
  <a class="pure-menu-heading brand" href="/"><img src="/img/ciheam.png" alt="CIHEAM" /></a>



  <div class="pure-menu">
    <ul class="pure-menu-list">
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="/"><i class='fa fa-home fa-fw'></i>Home</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="/about/"><i class='fa fa-user fa-fw'></i>About</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="/teaching/"><i class='fa fa-university fa-fw'></i>Teaching</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="/research/"><i class='fa fa-area-chart fa-fw'></i>Research</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="/papers/"><i class='fa fa-newspaper-o fa-fw'></i>Publications</a>
      
        </li>
      
    </ul>
  </div>

  <div class="pure-menu social">
  <ul class="pure-menu-list">

    

    
    <li class="pure-menu-item">
      <a class="pure-menu-link" href="mailto:tristan.berchoux@gmail.com" target="_blank"><i class="fa fa-envelope fa-fw"></i>Email</a>
    </li>
    

    
    <li class="pure-menu-item">
      <a class="pure-menu-link" href="https://linkedin.com/in/tristanberchoux" target="_blank"><i class="fa fa-linkedin-square fa-fw"></i>LinkedIn</a>
    </li>
    

    
    <li class="pure-menu-item">
      <a class="pure-menu-link" href="https://twitter.com/TristanBerchoux" target="_blank"><i class="fa fa-twitter-square fa-fw"></i>Twitter</a>
    </li>
    

    

    

    
    <li class="pure-menu-item">
      <a class="pure-menu-link" href="https://www.researchgate.net/profile/Tristan_Berchoux" target="_blank"><i class="fa fa-flask fa-fw"></i>ResearchGate</a>
    </li>
    

    
    <li class="pure-menu-item">
      <a class="pure-menu-link" href="https://scholar.google.co.uk/citations?user=shdhPjcAAAAJ%26hl%3den" target="_blank"><i class="fa fa-graduation-cap fa-fw"></i>Scholar</a>
    </li>
    


    
    <li class="pure-menu-item">
      <a class="pure-menu-link" href="http://orcid.org/0000-0003-4095-2164" target="_blank"><i class="fa fa-id-badge fa-fw"></i>ORCID</a>
    </li>
    


    


  </ul>
</div>


  <div>
  <div class="small-print">
    <small>&copy; 2019 by Tristan Berchoux</small>
  </div>
  <div class="small-print">
    <small>Built with <a href="https://github.com/rstudio/blogdown" target="_blank">blogdown</a> and <a href="https://gohugo.io/" target="_blank">Hugo</a>. Theme <a href="https://github.com/yoshiharuyamashita/blackburn" target="_blank">Blackburn</a>.</small>
  </div>
</div>

</div>


  <div id="main">


<div class="header">
  <h1>Input on the Draft NIH Strategic Plan for Data Science</h1>
  <h2></h2>
</div>
<div class="content">

  <div class="post-meta">

  <div>
    
    <i class="fa fa-user fa-fw"></i>
    <span>Rafael Irizarry</span>
    
    <i class="fa fa-calendar fa-fw"></i>
    <time>2018/04/02</time>
  </div>

  

  

  

</div>


  

<p>The NIH is soliciting input for their <a href="https://grants.nih.gov/grants/rfi/NIH-Strategic-Plan-for-Data-Science.pdf">Strategic Plan for Data Science</a>. If you are interested, today, April 2, is the deadline. You can provide input <a href="https://grants.nih.gov/grants/rfi/rfi.cfm?ID=73">here</a>. Below is what I plan to  submit.</p>

<h2 id="summary">Summary</h2>

<p>My main critique is that the report is somewhat vague. More specifics and concrete examples should be included.</p>

<p>My main concern is that the draft describes initiatives with the goal of improving the <em>back end</em> of data science (data storage, data management, and computing infrastructure) without realizing that to do this one needs to understand the needs of those working on the <em>front end</em> of data science (data exploration, quality assessment, <a href="https://simplystatistics.org/2017/04/03/interactive-data-analysis/">interactive data analysis</a>, and method development).</p>

<p>My main recommendation is that the NIH Data Science leadership complement their current expertise with individuals  experienced with the <em>front end</em> of data science. The ideal candidate has</p>

<ul>
<li>experienced the pain of explaining to a collaborator how to prepare a spreadsheet that is appropriate for data analysis,</li>
<li>performed interactive data analysis in R, Python or Matlab,</li>
<li>developed a software package used by researchers that are not lab members or collaborators,</li>
<li>has downloaded, imported and visually explored a publicly available dataset, <strong>and</strong> (not or)</li>
<li>has fitted a model or trained a machine learning algorithm to data.</li>
</ul>

<p>These individuals will have a very good sense of the specifics of what is required from a common architecture and infrastructure for it to facilitate successful data analysis and method development.</p>

<p>Below I include more detailed comments.</p>

<h2 id="the-perspective-of-the-interactive-data-analysts">The perspective of the interactive data analysts</h2>

<p>The draft defines Data Science as  &ldquo;the interdisciplinary field of inquiry in which quantitative and analytical approaches, processes, and systems are developed and used to extract knowledge and insights from increasingly large and/or complex sets of data&rdquo;.  Here I provide the perspective of a researcher that develops <em>analytical approaches</em> and performs, or supervises, the primary data analysis of a collaborative project interested in &ldquo;[extracting] knowledge and insights from increasingly large and/or complex sets of data&rdquo;. Note that I do not consider myself to be unique in this regard. Rather, I am part of a large community of statisticians and computational biologists with many commonalities in the way we work, the way we share the products of our work, and the way we depend and interact with the NIH.</p>

<h2 id="in-genomics-nih-already-does-a-great-job-of-supporting-data-science">In genomics, NIH already does a great job of supporting data science</h2>

<p>By funding numerous investigator initiated grants, the NIH has fostered a competitive environment that has served as a driving force for innovation and resulted in an impressive arsenal of data analysis and data processing methods. The NIH has also provided funding for the maintenance and improvement of software implementation of these ideas. My work, and the work of many others in my community, depend on these NIH funded tools and infrastructure. Examples include Bioconductor and its many packages, Bedtools, SAMtools, Bowtie, Recount, and IGV, just to name a few. We also depend on NIH  funded and organized resources such as the Short Read Archive (SRA), the Gene Expression Omnibus (GEO), and RefSeq as well as NIH datasets such as those produced by GTEX, TCGA, and ENCODE, again, just to name a few.</p>

<h2 id="what-can-improve">What can improve?</h2>

<p>The main pain points in my career have been</p>

<ol>
<li>The sub-optimal sample annotations provided by GEO and SRA.</li>
<li>The bureaucratic burden imposed by dbGap.</li>
<li>That lack of access to paired GWAS genotype and phenotype data, electronic medical record data, cohort studies, and clinical trials data.</li>
<li>The fact that often (not always!) data generators lack computational sophistication and basic statistical knowledge.</li>
</ol>

<h2 id="my-favorite-parts-of-the-draft">My favorite parts of the draft</h2>

<p>I was happy to see that the draft includes a proposal to separate funding mechanisms for data generation from the development of data analysis tools. This will indeed be a great improvement as it will give more independent groups with expertise in this arena an opportunity to compete. Currently too much of the data analysis funding goes to data generators which tend to use these to fund inexperienced trainees. This is particularly true of the large consortia.</p>

<p>I was also happy to see that the draft includes a proposal to better support current successful developers to &ldquo;transform, or harden, innovative algorithms and tools created by academic scientists into enterprise-ready resources that meet industry standards of ease of use and efficiency of operation.&rdquo;</p>

<p>I was most happy about the proposal to increase support for training and education.</p>

<h2 id="avoid-top-down-approaches-when-deciding-on-a-common-architecture-infrastructure-and-set-of-tools">Avoid top-down approaches when deciding on a &ldquo;common architecture, infrastructure, and set of tools&rdquo;</h2>

<p>In my view, the two main reasons the current ecosystem has been successful at producing data analysis tools are:</p>

<ol>
<li><p>Many groups are funded and permitted to compete for users of the data analysis workflows they produce.  Which workflows are best is decided by an unplanned process similar to a free market that includes independent assessment, productive debates, and sharing of experiences. Satisfied users translate into increased citations and gains in reputation, which in turn improve ones chances of promotion and funding.</p></li>

<li><p>Data is made publicly available as they come off the measurement instruments, what we refer to as <em>raw data</em>. In my experience, when a bureaucratic process is used to decide data formats, important information is often left out.</p></li>
</ol>

<p>My <strong>nightmare scenario</strong> is that the NIH stops making the raw data available and computationally accessible (say via secure copy or wget) and that instead we are forced to log on to a server on &ldquo;the cloud&rdquo; and access only processed data.</p>

<h2 id="we-already-follow-fair-principles">We already follow FAIR principles</h2>

<p>By using existing open source tools many of us are already following the Findable, Accessible, Interoperable, and Reusable (FAIR) principles. Achieving this took us years and we built on the shoulder of open source giants. We build these as a solution to make the way we analyze data for a variety of diverse projects more efficient. It was not developed before the front end data analytical challenges were understood since it would have been impossible to predict what would be appropriate.</p>

<p>My institute provides enough computing and data storage resource to download raw data when needed. We also store data from our collaborators. Using a Linux file system and best practices for file management we organize the raw data in a way that is shareable and findable among those with access to our system. File system permissions are used to assure data is only accessed with those with permission to access it. We use UNIX shell commands or Python to organize workflows based on open source command-line tools, Bowtie for example, to convert raw data into formats we have determined to be sufficient for front end analysis. These workflows are constantly changing, but the shell or Python scripts permit reproducibility  and back compatibility. Interactive data analysis is done with open source data analysis software such as R and Python. Tools such as Biocondoctor implement object oriented programming to ensure interoperability and reusability across data from different technologies. R packages from the tidyverse also make use of interoperability and reusability concepts for downstream analysis and the generation of figures. We use git for version control and GitHub to share code publicly.</p>

<p>The NIH should not assume that they can do better by designing a new approach from first principles and making decisions by committee. One really needs to walk in the shoes of a data analysts to do this right. I understand that the above approach will not work for everybody since it requires much institutional support as well as expertise. But we do recommend against top-down approaches to solving this problem. We recommend that that independent groups compete for funding and once funded compete for users.</p>


  
<div class="prev-next-post pure-g">
  <div class="pure-u-1-24" style="text-align: left;">
    
    <a href="/2018/02/08/what-do-fahrenheit-comma-separated-files-and-markdown-have-in-common/"><i class="fa fa-chevron-left"></i></a>
    
  </div>
  <div class="pure-u-10-24">
    
    <nav class="prev">
      <a href="/2018/02/08/what-do-fahrenheit-comma-separated-files-and-markdown-have-in-common/">What do Fahrenheit, comma separated files, and markdown have in common? </a>
    </nav>
    
  </div>
  <div class="pure-u-2-24">
    &nbsp;
  </div>
  <div class="pure-u-10-24">
    
    <nav class="next">
      <a href="/2018/04/17/what-is-a-successful-data-analysis/">What is a Successful Data Analysis?</a>
    </nav>
    
  </div>
  <div class="pure-u-1-24" style="text-align: right;">
    
    <a href="/2018/04/17/what-is-a-successful-data-analysis/"><i class="fa fa-chevron-right"></i></a>
    
  </div>
</div>



  


</div>

</div>
</div>
<script src="/js/ui.js"></script>
<script src="//yihui.name/js/math-code.js"></script>
<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true
  }
});
</script>
<script async src="//cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML"></script>




</body>
</html>

