<!DOCTYPE html>
<html lang="en">

<head>
  <meta http-equiv="content-type" content="text/html; charset=utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="description" content="">
  <meta name="generator" content="Hugo 0.54.0" />

  <title>Tristan Berchoux</title>

  
  
  <link rel="stylesheet" href="https://cdn.bootcss.com/pure/0.6.2/pure-min.css">

  <!--[if lte IE 8]>
  <link rel="stylesheet" href="https://cdn.bootcss.com/pure/0.6.2/grids-responsive-old-ie-min.css">
  <![endif]-->
  <!--[if gt IE 8]><!-->
  <link rel="stylesheet" href="https://cdn.bootcss.com/pure/0.6.2/grids-responsive-min.css">
  <!--<![endif]-->

  <!--[if lte IE 8]>
  <link rel="stylesheet" href="/css/side-menu-old-ie.css">
  <![endif]-->
  <!--[if gt IE 8]><!-->
  <link rel="stylesheet" href="/css/side-menu.css">
  <!--<![endif]-->

  <link rel="stylesheet" href="/css/blackburn.css">

  <link rel="stylesheet" href="/css/github-gist.css" rel="stylesheet" id="theme-stylesheet">
  <script src="/js/highlight.pack.js"></script>
  <script>hljs.initHighlightingOnLoad();</script>

  
  <script async src="https://use.fontawesome.com/32c3d13def.js"></script>

  
  
  <link rel="alternate" type="application/rss+xml" title="Tristan Berchoux" href="/index.xml" />
  

  
  <link rel="stylesheet" href="//cdn.bootcss.com/highlight.js/9.11.0/styles/github.min.css">
  <script src="//cdn.bootcss.com/highlight.js/9.11.0/highlight.min.js"></script>
  <script src="//cdn.bootcss.com/highlight.js/9.11.0/languages/r.min.js"></script>
  <script>hljs.configure({languages: []}); hljs.initHighlightingOnLoad();</script>
  

  <link rel="shortcut icon" href="/img/favicon.ico" type="image/x-icon" />

  
  

</head>


<body>
<div id="layout">

  
<a href="#menu" id="menuLink" class="menu-link">
  
  <span></span>
</a>
<div id="menu">

  
  <a class="pure-menu-heading brand" href="/"><img src="/img/ciheam.png" alt="CIHEAM" /></a>



  <div class="pure-menu">
    <ul class="pure-menu-list">
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="/"><i class='fa fa-home fa-fw'></i>Home</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="/about/"><i class='fa fa-user fa-fw'></i>About</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="/teaching/"><i class='fa fa-university fa-fw'></i>Teaching</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="/research/"><i class='fa fa-area-chart fa-fw'></i>Research</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="/papers/"><i class='fa fa-newspaper-o fa-fw'></i>Publications</a>
      
        </li>
      
    </ul>
  </div>

  <div class="pure-menu social">
  <ul class="pure-menu-list">

    

    
    <li class="pure-menu-item">
      <a class="pure-menu-link" href="mailto:tristan.berchoux@gmail.com" target="_blank"><i class="fa fa-envelope fa-fw"></i>Email</a>
    </li>
    

    
    <li class="pure-menu-item">
      <a class="pure-menu-link" href="https://linkedin.com/in/tristanberchoux" target="_blank"><i class="fa fa-linkedin-square fa-fw"></i>LinkedIn</a>
    </li>
    

    
    <li class="pure-menu-item">
      <a class="pure-menu-link" href="https://twitter.com/TristanBerchoux" target="_blank"><i class="fa fa-twitter-square fa-fw"></i>Twitter</a>
    </li>
    

    

    

    
    <li class="pure-menu-item">
      <a class="pure-menu-link" href="https://www.researchgate.net/profile/Tristan_Berchoux" target="_blank"><i class="fa fa-flask fa-fw"></i>ResearchGate</a>
    </li>
    

    
    <li class="pure-menu-item">
      <a class="pure-menu-link" href="https://scholar.google.co.uk/citations?user=shdhPjcAAAAJ%26hl%3den" target="_blank"><i class="fa fa-graduation-cap fa-fw"></i>Scholar</a>
    </li>
    


    
    <li class="pure-menu-item">
      <a class="pure-menu-link" href="http://orcid.org/0000-0003-4095-2164" target="_blank"><i class="fa fa-id-badge fa-fw"></i>ORCID</a>
    </li>
    


    


  </ul>
</div>


  <div>
  <div class="small-print">
    <small>&copy; 2019 by Tristan Berchoux</small>
  </div>
  <div class="small-print">
    <small>Built with <a href="https://github.com/rstudio/blogdown" target="_blank">blogdown</a> and <a href="https://gohugo.io/" target="_blank">Hugo</a>. Theme <a href="https://github.com/yoshiharuyamashita/blackburn" target="_blank">Blackburn</a>.</small>
  </div>
</div>

</div>


  <div id="main">


<div class="header">
  <h1>Tristan Berchoux</h1>
  <h2>Spatial Planning • Economic Geography • Rural Development</h2>
</div>

<div class="content">
  
    <article>
  <header>
    <h2><a href="/2014/04/07/writing-good-software-can-have-more-impact-than-publishing-in-high-impact-journals-for-genomic-statisticians/">Writing good software can have more impact than publishing in high impact journals for genomic statisticians</a></h2>

    <div class="post-meta">

  <div>
    
    <i class="fa fa-user fa-fw"></i>
    <span>Rafael Irizarry</span>
    
    <i class="fa fa-calendar fa-fw"></i>
    <time>2014/04/07</time>
  </div>

  

  

  

</div>


  </header>

  <p>
  Every once in a while we see computational papers published in science journals with high impact factors. Genomics related methods appear quite often in these journals. Several of my junior colleagues express frustration that all their papers get rejected from these journals. I tell them that the same is true for most of my papers and remind them of these examples:
  Method   Journal   Year   #Citations     PLINK   AJHG  2007  6481     Bioconductor   Genome Biology  2004  5973     RMA   Biostatistics  2003  5674     limma   SAGMB  2004  5637     quantile normalization   Bioinformatics  2003  4646     Bowtie   Genome Biology  2009  3849     BWA   Bioinformatics  2009  3327     Loess normalization   NAR  2002  3313     qvalues   JRSS-B  2002  2758     tophat   Bioinformatics  2008  1868     vsn   Bioinformatics  2002  1398     GCRMA   JASA  2004  1397     MACS   Genome Biology  2008  1277     deseq   Genome Biology  2010  1264     CBS   Biostatistics  2004  1051     R/qtl   Bioinformatics  2003  1027    Let me know of other examples in the comments.
  </p>

  
  <footer>
    <a href="/2014/04/07/writing-good-software-can-have-more-impact-than-publishing-in-high-impact-journals-for-genomic-statisticians/">Read more<i class="fa fa-angle-double-right fa-fw"></i></a>
  </footer>
  
</article>

  
    <article>
  <header>
    <h2><a href="/2014/04/01/this-is-how-an-important-scientific-debate-is-being-used-to-stop-epa-regulation/">This is how an important scientific debate is being used to stop EPA regulation</a></h2>

    <div class="post-meta">

  <div>
    
    <i class="fa fa-user fa-fw"></i>
    <span>Roger Peng</span>
    
    <i class="fa fa-calendar fa-fw"></i>
    <time>2014/04/01</time>
  </div>

  

  

  

</div>


  </header>

  <p>
  Environmental regulation in the United States has protected human health for over 40 years. Since the Clean Air Act was enacted in 1970, levels of outdoor air pollution have dropped dramatically, changing the landscape of once heavily-polluted cities like Los Angeles and Pittsburgh. A 2011 cost-benefit analysis conducted by the U.S. Environmental Protection Agency estimated that the 1990 amendments to the CAA prevented 160,000 deaths and 13 million lost work days in the year 2010 alone.
  </p>

  
  <footer>
    <a href="/2014/04/01/this-is-how-an-important-scientific-debate-is-being-used-to-stop-epa-regulation/">Read more<i class="fa fa-angle-double-right fa-fw"></i></a>
  </footer>
  
</article>

  
    <article>
  <header>
    <h2><a href="/2014/03/31/data-analysis-for-genomic-edx-course/">Data Analysis for Genomics edX Course</a></h2>

    <div class="post-meta">

  <div>
    
    <i class="fa fa-user fa-fw"></i>
    <span>Rafael Irizarry</span>
    
    <i class="fa fa-calendar fa-fw"></i>
    <time>2014/03/31</time>
  </div>

  

  

  

</div>


  </header>

  <p>
  Mike Love (@mikelove) and I have been working hard the past couple of months preparing a free online edX course on data analysis for genomics. Our target audience are the postdocs, graduate students and research scientists that are tasked with analyzing genomics data, but don&rsquo;t have any formal training. The eight week course will start with the very basics, but will ramp up rather quickly and end with real-life workflows for genome variation, RNA-seq, DNA methylation, and ChIP-seq.
  </p>

  
  <footer>
    <a href="/2014/03/31/data-analysis-for-genomic-edx-course/">Read more<i class="fa fa-angle-double-right fa-fw"></i></a>
  </footer>
  
</article>

  
    <article>
  <header>
    <h2><a href="/2014/03/26/a-non-comprehensive-comparison-of-prominent-data-science-programs-on-cost-and-frequency/">A non-comprehensive comparison of prominent data science programs on cost and frequency.</a></h2>

    <div class="post-meta">

  <div>
    
    <i class="fa fa-user fa-fw"></i>
    <span>Jeff Leek</span>
    
    <i class="fa fa-calendar fa-fw"></i>
    <time>2014/03/26</time>
  </div>

  

  

  

</div>


  </header>

  <p>
  We did a really brief comparison of a few notable data science
programs for a grant submission we were working on. I thought it was pretty fascinating, so I&rsquo;m posting it here. A couple of notes about the table.
 Our program can be taken for free, which includes assessments. If you want the official certificate and to take the capstone you pay the above costs.
 Udacity&rsquo;s program can also be taken for free, but if you want the official certificate, assessments, or tutoring you pay the above costs.
  </p>

  
  <footer>
    <a href="/2014/03/26/a-non-comprehensive-comparison-of-prominent-data-science-programs-on-cost-and-frequency/">Read more<i class="fa fa-angle-double-right fa-fw"></i></a>
  </footer>
  
</article>

  
    <article>
  <header>
    <h2><a href="/2014/03/24/the-fact-that-data-analysts-base-their-conclusions-on-data-does-not-mean-they-ignore-experts/">The fact that data analysts base their conclusions on data does not mean they ignore experts</a></h2>

    <div class="post-meta">

  <div>
    
    <i class="fa fa-user fa-fw"></i>
    <span>Rafael Irizarry</span>
    
    <i class="fa fa-calendar fa-fw"></i>
    <time>2014/03/24</time>
  </div>

  

  

  

</div>


  </header>

  <p>
  Paul Krugman recently joined the new FiveThirtyEight hating bandwagon. I am not crazy about the new website either (although I&rsquo;ll wait more than one weeks before judging) but in a recent post Krugman creates a false dichotomy that is important to correct. Krugmanam states that &ldquo;[w]hat [Nate Silver] seems to have concluded is that there are no experts anywhere, that a smart data analyst can and should ignore all that.&rdquo; I don&rsquo;t think that is what Nate Silver, nor any other smart data scientist or applied statistician has concluded.
  </p>

  
  <footer>
    <a href="/2014/03/24/the-fact-that-data-analysts-base-their-conclusions-on-data-does-not-mean-they-ignore-experts/">Read more<i class="fa fa-angle-double-right fa-fw"></i></a>
  </footer>
  
</article>

  
    <article>
  <header>
    <h2><a href="/2014/03/20/the-8020-rule-of-statistical-methods-development/">The 80/20 rule of statistical methods development</a></h2>

    <div class="post-meta">

  <div>
    
    <i class="fa fa-user fa-fw"></i>
    <span>Jeff Leek</span>
    
    <i class="fa fa-calendar fa-fw"></i>
    <time>2014/03/20</time>
  </div>

  

  

  

</div>


  </header>

  <p>
  Developing statistical methods is hard and often frustrating work. One of the under appreciated rules in statistical methods development is what I call the 80&frasl;20 rule (maybe could even by the 90&frasl;10 rule). The basic idea is that the first reasonable thing you can do to a set of data often is 80% of the way to the optimal solution. Everything after that is working on getting the last 20%. (_Edit: Rafa points out that once again I&rsquo;ve reverse-scooped a bunch of people and this is already a thing that has been pointed out many times.
  </p>

  
  <footer>
    <a href="/2014/03/20/the-8020-rule-of-statistical-methods-development/">Read more<i class="fa fa-angle-double-right fa-fw"></i></a>
  </footer>
  
</article>

  
    <article>
  <header>
    <h2><a href="/2014/03/19/the-end-of-the-world-challenge/">The time traveler&#39;s challenge.</a></h2>

    <div class="post-meta">

  <div>
    
    <i class="fa fa-user fa-fw"></i>
    <span>Jeff Leek</span>
    
    <i class="fa fa-calendar fa-fw"></i>
    <time>2014/03/19</time>
  </div>

  

  

  

</div>


  </header>

  <p>
  Editor&rsquo;s note: This has nothing to do with statistics. 
I do a lot of statistics for a living and would claim to know a relatively large amount about it. I also know a little bit about a bunch of other scientific disciplines, a tiny bit of engineering, a lot about pointless sports trivia, some current events, the geography of the world (vaguely) and the geography of places I&rsquo;ve lived (pretty well).
  </p>

  
  <footer>
    <a href="/2014/03/19/the-end-of-the-world-challenge/">Read more<i class="fa fa-angle-double-right fa-fw"></i></a>
  </footer>
  
</article>

  
    <article>
  <header>
    <h2><a href="/2014/03/14/enar-is-in-baltimore-heres-what-to-do/">ENAR is in Baltimore - Here&#39;s What To Do</a></h2>

    <div class="post-meta">

  <div>
    
    <i class="fa fa-user fa-fw"></i>
    <span>Roger Peng</span>
    
    <i class="fa fa-calendar fa-fw"></i>
    <time>2014/03/14</time>
  </div>

  

  

  

</div>


  </header>

  <p>
  This year&rsquo;s meeting of the Eastern North American Region of the International Biometric Society (ENAR) is in lovely Baltimore, Maryland. As local residents Jeff and I thought we&rsquo;d put down a few suggestions for what to do during your stay here in case you&rsquo;re not familiar with the area.
Venue
The conference is being held at the Marriott in the Harbor East area of the city, which is relatively new and a great location.
  </p>

  
  <footer>
    <a href="/2014/03/14/enar-is-in-baltimore-heres-what-to-do/">Read more<i class="fa fa-angle-double-right fa-fw"></i></a>
  </footer>
  
</article>

  
    <article>
  <header>
    <h2><a href="/2014/03/14/using-bioconductor-to-find-some-empirical-evidence-in-support-of-%CF%80-being-a-normal-number/">How to use Bioconductor to find empirical evidence in support of π being a normal number</a></h2>

    <div class="post-meta">

  <div>
    
    <i class="fa fa-user fa-fw"></i>
    <span>Rafael Irizarry</span>
    
    <i class="fa fa-calendar fa-fw"></i>
    <time>2014/03/14</time>
  </div>

  

  

  
  
  
  <div>
    <i class="fa fa-tags fa-fw"></i>
    
      <a class="post-taxonomy-tag" href="/tags/r">R</a>
    
  </div>
  
  

</div>


  </header>

  <p>
  Happy π day everybody!
I wanted to write some simple code (included below) to the test parallelization capabilities of my new cluster. So, in honor of π day, I decided to check for evidence that π is a normal number. A normal number is a real number whose infinite sequence of digits has the property that picking any given random m digit pattern is 10−m. For example, using the Poisson approximation, we can predict that the pattern &ldquo;123456789&rdquo; should show up between 0 and 3 times in the first billion digits of π (it actually shows up twice starting, at the 523,551,502-th and 773,349,079-th decimal places).
  </p>

  
  <footer>
    <a href="/2014/03/14/using-bioconductor-to-find-some-empirical-evidence-in-support-of-%CF%80-being-a-normal-number/">Read more<i class="fa fa-angle-double-right fa-fw"></i></a>
  </footer>
  
</article>

  
    <article>
  <header>
    <h2><a href="/2014/03/12/oh-no-the-leekasso/">Oh no, the Leekasso....</a></h2>

    <div class="post-meta">

  <div>
    
    <i class="fa fa-user fa-fw"></i>
    <span>Jeff Leek</span>
    
    <i class="fa fa-calendar fa-fw"></i>
    <time>2014/03/12</time>
  </div>

  

  

  

</div>


  </header>

  <p>
  An astute reader (Niels Hansen, who is visiting our department today) caught a bug in my code on Github for the Leekasso. I had:
lm1 = lm(y ~ leekX)
predict.lm(lm1,as.data.frame(leekX2))
Unfortunately, this meant that I was getting predictions for the training set on the test set. Since I set up the test/training sets the same, this meant that I was actually getting training set error rates for the Leekasso. Neils Hansen noticed the bug and reran the fixed code with this term instead:
  </p>

  
  <footer>
    <a href="/2014/03/12/oh-no-the-leekasso/">Read more<i class="fa fa-angle-double-right fa-fw"></i></a>
  </footer>
  
</article>

  

  


<nav class="pagination" role="pagination">
  
  <a href="/page/34/"><i class="fa fa-chevron-left"></i></a>
  
  <span>&nbsp;35 / 104&nbsp;</span>
  
  <a href="/page/36/"><i class="fa fa-chevron-right"></i></a>
  
</nav>



</div>

</div>
</div>
<script src="/js/ui.js"></script>
<script src="//yihui.name/js/math-code.js"></script>
<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true
  }
});
</script>
<script async src="//cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML"></script>




</body>
</html>

